airqo-platform/AirQo-api
# Ft locate update

- What does this PR do?
This PR refactors code for the locate microservice.
- What JIRA task is related to this PR?
N/A
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/locate`. Run py -m venv env to create a python virtual environment. Run env\scripts\activate to activate the environment and then run pip install -r requirements.txt to install necessary packages. Afterwards, run `flask run` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
Base url: http:localhost:4001/api/v1
Partial urls: 
1. /map/parishes
2. /map/savelocatemap
3. map/getlocatemap/<user_id>
4. map/updatelocatemap/<SPACE_NAME>
5. map/deletelocatemap/<SPACE_NAME>
More in the [api documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=562835484)
